<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>照片处理器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>image/icons/设备在线配置-2.svg</normaloff>image/icons/设备在线配置-2.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="ScrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>421</width>
      <height>531</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>419</width>
       <height>529</height>
      </rect>
     </property>
     <widget class="QLabel" name="LabPic">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>401</width>
        <height>491</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="SavedFileBtn">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>保存为</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SavedFileName">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="ImgSlider">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>0</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>68</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>图片质量：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ImgSliderVal">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>0</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="DockWid">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="DockWidContents">
    <widget class="QTreeWidget" name="PhotoTreeWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>521</width>
       <height>551</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>节点</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>节点类型</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="MainToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="SecondToolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <action name="actAddFolder">
   <property name="text">
    <string>添加组合</string>
   </property>
  </action>
  <action name="actAddFiles">
   <property name="text">
    <string>添加图片</string>
   </property>
  </action>
  <action name="actDeleteItem">
   <property name="text">
    <string>删除节点</string>
   </property>
  </action>
  <action name="actScanItems">
   <property name="text">
    <string>遍历节点</string>
   </property>
  </action>
  <action name="actZoomFitH">
   <property name="text">
    <string>自适应高度</string>
   </property>
  </action>
  <action name="actZoomFitW">
   <property name="text">
    <string>自适应宽度</string>
   </property>
  </action>
  <action name="actZoomIn">
   <property name="text">
    <string>放大显示</string>
   </property>
  </action>
  <action name="actZoomRealSize">
   <property name="text">
    <string>真实显示</string>
   </property>
  </action>
  <action name="actDockVisible">
   <property name="text">
    <string>停靠区可见</string>
   </property>
  </action>
  <action name="actDockFloat">
   <property name="text">
    <string>停靠区浮动性</string>
   </property>
   <property name="toolTip">
    <string>停靠区浮动性</string>
   </property>
  </action>
  <action name="actZoomDe">
   <property name="text">
    <string>缩小显示</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>退出程序</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
